specification/compute/resource-manager/Microsoft.Compute/stable/2021-08-01/disk.json

```diff
@@ -3482,7 +3485,17 @@
         },
         "completionPercent": {
           "type": "number",
-          "description": "Percentage complete for the background copy when a resource is created via the CopyStart operation."
+          "description": "Percentage complete for the background copy of disk restore point when source resource is from a different region."
+        },
+        "replicationState": {
+          "type": "string",
+          "description": "Replication state of disk restore point when source resource is from a different region.",
+          "readOnly": true
+        },
+        "sourceResourceLocation": {
+          "type": "string",
+          "description": "Location of source disk or source disk restore point when source resource is from a different region.",
```

Comment:
Is the field 'sourceResourceLocation' returned only when source resource is from a different region? Pls. validate, else you may have to update the description as well as the example of local disk restore point

Reply:
yes it is returned only for remote scenarios